File: add/codesnippet/CSharp/7220a29d-c2cf-4eb5-a772-_1.cs
        public void DrawRectangleRectangle(PaintEventArgs e)
        {
                     
            // Create pen.
            Pen blackPen = new Pen(Color.Black, 3);
                     
            // Create rectangle.
            Rectangle rect = new Rectangle(0, 0, 200, 200);
                     
            // Draw rectangle to screen.
            e.Graphics.DrawRectangle(blackPen, rect);
        }

File: add/codesnippet/CPP/7220a29d-c2cf-4eb5-a772-_1.cpp
public:
   void DrawRectangleRectangle( PaintEventArgs^ e )
   {
      // Create pen.
      Pen^ blackPen = gcnew Pen( Color::Black,3.0f );

      // Create rectangle.
      Rectangle rect = Rectangle(0,0,200,200);

      // Draw rectangle to screen.
      e->Graphics->DrawRectangle( blackPen, rect );
   }